packages/cashier/src/Components/side-note.jsx

```diff
@@ -1,39 +1,80 @@
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import React from 'react';
 import { DesktopWrapper, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
 import 'Sass/side-note.scss';
 
-const SideNoteBullet = ({ children }) => (
+const SideNoteTitle = ({ side_notes, title }) => (
+    <Text className='side-note__title' weight='bold' as='p'>
+        {title ||
+            (side_notes?.length > 1 ? <Localize i18n_default_text='Notes' /> : <Localize i18n_default_text='Note' />)}
+    </Text>
+);
+
+const SideNoteText = ({ children }) => (
+    <Text className='side-note__text' size='xxs' as='p'>
+        {children}
+    </Text>
+);
+
+const SideNoteBullet = ({ children, component }) => (
     <div className='side-note__bullet-wrapper'>
         <div className='side-note__bullet' />
-        <span>{children}</span>
+        {component ? children : <SideNoteText>{children}</SideNoteText>}
     </div>
 );
 
-const SideNote = ({ notes, title, has_bullets }) => (
-    <div className='side-note'>
-        {title && (
-            <DesktopWrapper>
-                <Text className='side-note__text' weight='bold' as='p'>
-                    {title}
-                </Text>
-            </DesktopWrapper>
-        )}
-
-        {notes.map((note, i) =>
-            has_bullets ? (
-                <SideNoteBullet key={i}>{note}</SideNoteBullet>
-            ) : (
-                <Text key={i} className='side-note__text' size='xxs' as='p'>
-                    {note}
-                </Text>
-            )
-        )}
-    </div>
-);
+const SideNote = ({ side_notes, title, has_bullets = true, className }) => {
+    const checkNote = (note, i) => {
+        let Component;
+        if (typeof note === 'string') {
+            if (has_bullets)
+                return (
+                    <SideNoteBullet key={i}>
+                        <Localize i18n_default_text={note} />
+                    </SideNoteBullet>
+                );
+            return <SideNoteText key={note.key || i}>{note.i18n_default_text}</SideNoteText>;
+        } else if (typeof note === 'object' && note.props.i18n_default_text) {
```

Comment:
Why are you checking `note.props.i18n_default_text`?

Reply:
I check this prop for determining what is the component. If it has `i18n_default_text` prop, so this is `Localize` component which can have bullet, if not, bullet won't be checked for that and the component will be returned without any process.
It would be better if there is any way to determine the name of the component, but I couldn't find one.